src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -137,12 +137,6 @@ public static class WebScriptHostBuilderExtension
                     ConfigureRegisteredBuilders(services, rootServiceProvider);
                 });
 
-            var debugStateProvider = rootServiceProvider.GetService<IDebugStateProvider>();
-            if (!FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagDisableDevInDebug, environment) && debugStateProvider.InDebugMode)
```

Comment:
Can you remove the `FeatureFlagDisableDevInDebug` constant?

Reply:
done